.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Request a feature
+    url: http://cool.haxx.se/cgi-bin/mailman/listinfo/curl-library
+    about: This is only for bugs! Feature requests should go to the ML
```

Comment:
I would spell it out as mailing list instead of ML so that newcomers understand it better.

Reply:
Maybe the link should also be this page instead: https://curl.haxx.se/mail/